changelog/split-std-datetime.dd

```diff
@@ -0,0 +1,43 @@
+std.datetime has been split into a package.
+
+std.datetime is now a package containing the following modules:
+
+$(UL
+    $(LI <a href="$(PHOBOS_PATH)std_datetime_common.html">std.datetime.common</a>)
+    $(LI <a href="$(PHOBOS_PATH)std_datetime_date.html">std.datetime.date</a>)
+    $(LI <a href="$(PHOBOS_PATH)std_datetime_datetime.html">std.datetime.datetime</a>)
+    $(LI <a href="$(PHOBOS_PATH)std_datetime_interval.html">std.datetime.interval</a>)
+    $(LI <a href="$(PHOBOS_PATH)std_datetime_systime.html">std.datetime.systime</a>)
+    $(LI <a href="$(PHOBOS_PATH)std_datetime_timeofday.html">std.datetime.timeofday</a>)
+    $(LI <a href="$(PHOBOS_PATH)std_datetime_timezone.html">std.datetime.timezone</a>)
+)
+
+$(LI <a href="$(PHOBOS_PATH)std_datetime.html">std.datetime.package</a>)
+publicly imports all of those modules. So, it should be the case that no
+existing code will break, as everything in std.datetime will still be imported
+by importing std.datetime. New code can choose to import the modules
+individually or to import the entire package.
+
+std.datetime.common contains a few types and free functions that are used by
+the other modules (such as Month and isTimePoint).
```

Comment:
I'm going to say that:
1. I think e.g. std.datetime.common should really be a link to the file
2. I have no idea how that link should be written.

Is there a way this can be fixed after the merge?